Resources/Prototypes/Entities/Objects/Weapons/Guns/SMGs/smgs.yml

```
@@ -181,7 +181,7 @@
   name: WT550
   parent: [ BaseWeaponSubMachineGun, BaseSecurityContraband ]
   id: WeaponSubMachineGunWt550
-  description: An excellent SMG, produced by NanoTrasen's Small Arms Division. Uses .35 auto ammo.
+  description: A truly unique firearm, the WT550 is the result of an Nanotrasen program to create a compact submachine gun fully controllable with one hand. It contains an exotic internal recoil buffer and feeds from special top-mounted .35 magazines. Manufacturing setbacks and a high unit cost make it rarely seen outside the hands of special forces.
```

This sentence is still a problem, for the reasons stated previously. I would personally omit it as the last sentence is a good ending.

```suggestion
  description: A truly unique firearm, the WT550 is the result of an Nanotrasen program to create a compact submachine gun fully controllable with one hand. It contains an exotic internal recoil buffer and feeds from special top-mounted .35 magazines.
```

---

Okay I swear in the document that part was trimmed as you suggested, but I accidentally marked it updated without updating it